### PR TITLE
Add missing unit tests for package.py

### PR DESCRIPTION
**Pull request by sfc-gh-jsikorski**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [x] I've added new automated tests to verify correctness of my new code.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [x ] I've described my changes in the section below.

### Changes description
Added unit tests for package.py

### CODE REVIEW

File: tests/test_package.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+import pytest
+
+from snowcli.cli.snowpark import package
+
+from tests.test_data import test_data
+
+
+class TestPackage:
+    LOGGER = logging.getLogger(__name__)
```
Comment (sfc-gh-astus) [inline tests/test_package.py:16]: it is not needed

Comment (sfc-gh-jsikorski) [reply tests/test_package.py:16]: You`re right. It`s a sole survivor of a different approach :D

File: tests/test_package.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        mock_response.json.return_value = test_data.anaconda_response
+        mock_requests.get.return_value = mock_response
+
+        monkeypatch.setattr("sys.stdin", io.StringIO("N"))
+
+        with caplog.at_level(logging.DEBUG, logger=argument[2]):
+            result = package.package_lookup(argument[0], install_packages=True)
+
+        assert caplog.text
+        assert argument[1] in caplog.text
```
Comment (sfc-gh-astus) [inline tests/test_package.py:46]: assert logs is the only way to test it?

Comment (sfc-gh-jsikorski) [reply tests/test_package.py:44]: It felt, that functions in package.py rely on so many outside connections (anaconda, pip, etc.), that it makes more sense to test them in integration tests. But i'm open to new test case propositions.

File: tests/test_package.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            ),
+            (
+                "some-weird-package-we-dont-know",
+                "not found in Snowflake anaconda channel...",
+                "snowcli.utils",
+            ),
+        ],
+    )
+    @patch("tests.test_package.package.utils.requests")
+    def test_package_lookup(self, mock_requests, caplog, argument, monkeypatch):
```
Comment (sfc-gh-astus) [inline tests/test_package.py:32]: Lookup function has a lot of ifs, these 2 cases cover all of them?

File: tests/test_package.py (hunk 2 above)
Comment (sfc-gh-astus) [reply tests/test_package.py:44]: Don't see other way ☹️

File: tests/test_package.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from snowcli.cli.snowpark import package
+
+from tests.test_data import test_data
+
+
+class TestPackage:
```
Comment (sfc-gh-astus) [inline tests/test_package.py:15]: Tests for `package_upload` are missing

Comment (sfc-gh-jsikorski) [reply tests/test_package.py:15]: Yes, `package_upload` return type is None, and it`s purpose is in side-effect of uploading the zipped package to Snowflake.
Of course, i could mock every stage, but the test would be really checking if mocks return correct values - therefore i think it`s rather a candidate for an. integration test

**Review by sfc-gh-astus — APPROVED** [review]

**Pull request merged**